source/process/casestudy.rst

```diff
@@ -14,8 +14,15 @@ The process is approximately as follows:
 1. Customer has completed a deployment of Mattermost and is open to a case study. 
 2. Customer case study contact is introduced to a Mattermost case study coordinator over email and completes a questionnaire (below). 
 3. Mattermost case study coordinator arranges a 30-minute phone or video call with the customer for a discussion around the questionnaire and deployment. 
-4. One to two weeks later a draft case study is sent to the customer for review and approval. 
-5. After customer approval, the case study is published to the Mattermost blog and may be used in other marketing materials to share with other potential customers the benefits of a Mattermost solution.
+4. One to two weeks later a draft case study is sent to the customer for review and approval (in a shared Google document).
+5. Mattermost case study coordinator:
+
+   -  politely follows up with the customer every 2 business days for an estimate of the draft case study review time
```

Comment:
Also, regarding @jasonblais question on why 2 days, propose we update so it's either checking in the day before the customer says they'll review, to confirm review is coming, or every 2 days if we don't have an estimated review date, just to keep up the energy. 

@amyblais I'd propose softening the language so 2 days is more a suggestion than a rule and coordinator should use their judgement.